apps/publisher/themes/default/js/list_assets.js

```diff
@@ -189,7 +189,9 @@ var createQuery = function(options) {
     var input = $('#inp_searchAsset').val();
     var category = options.category || undefined;
     var searchQueryString = '?';
-    input = setDefaultSearchQuery(input);
+    if (input.indexOf(":") == -1) {
```

Comment:
Your earlier logic was correct.In addition to reverting to the old state please add the following line before the condition check.
input = input.trim();